### ISSUE TITLE
Bootloop and Kernel Exception with Trace while Configuring Wireguard

### ISSUE THREAD

**Issue by skl283** [opnsense/src#46]
Hi guys,

i think i've found a serious Bug. I've added 2 Devices like this to get Wireguard to Work and to configure policy based routing (Copy & Paste out of the backup.xml)

```
    <opt3>
      <if>wg0</if>
      <descr>WGVPN</descr>
     <enable>1</enable>
      <lock>1</lock>
      <spoofmac/>
    </opt3>
    <opt4>
      <if>wg1</if>
      <descr>WGAZIRE</descr>
      <enable>1</enable>
      <lock>1</lock>
      <spoofmac/>
    </opt4>
```

and i've configured wireguard to use these Interfaces. While activating these Interfaces the System reboot.

You can look at the Stack-Trace which i've already sent by Mail through the webinterface. I've uploaded it here: https://pastebin.com/dsApfGcz

Are you having any hints or suggestions or could fix my issue? I've posted already a Thread in the OPNSense Forum, but did not get an answer.  Link: https://forum.opnsense.org/index.php?topic=14044.msg64586#msg64586

I need both Interfaces. One for VPN Incoming Connections, and the other for my VPN Provider. Is it possible to fix the crash? Could i change something to get rid of the bootloop?

**Comment by skl283** [comment]
Since October last year nothing happened. Are there any chances to get this issue fixed? Or do i need to test v20.1?

**Comment by AdSchellevis** [comment]
@skl283 policy based routing? it might be https://github.com/opnsense/src/issues/52, depending on configuration you might consider disabling "shared forwarding" in firewall advanced if it's the same issue
